tests/e2e/gc_cns_nodevm_attachment.go

```diff
@@ -68,24 +68,22 @@ var _ = ginkgo.Describe("[csi-guest] CnsNodeVmAttachment persistence", func() {
 		setResourceQuota(svcClient, svNamespace, defaultrqLimit)
 	})
 
-	/*
-		Test-1 (Attach/Detach)
-
-		Test Steps
-
-		Create a PVC using any replicated storage class from the SV.
-		Wait for PVC to be in Bound phase
-		Create a Pod with this PVC mounted as a volume
-		Verify CnsNodeVmAttachment CRD is created
-		Verify Pod is in Running phase
-		Verify volume is attached to VM.
-		Verify CnsNodeVmAttachment.Attached is true
-		Delete Pod
-		Verify CnsNodeVmAttachment CRD is deleted
-		Verify Pod is deleted from GC
-		Verify volume is detached from VM.
-		Delete PVC in GC
-	*/
+	// Test-1 (Attach/Detach)
```

Comment:
One query. Can't we use multi-line comment?

Reply:
Yes, you can. But, we need to be consistent in the way of comments. According to https://golang.org/doc/effective_go#commentary

"Line comments are the norm; block comments appear mostly as package comments, but are useful within an expression or to disable large swaths of code."

In this case, we are adding comments to a function. We should consistently use "//" like the rest of (product) code.